### ISSUE TITLE
Possibility to pick apples and pumpkins

### ISSUE THREAD

**Issue by ghost** [DayZMod/DayZ#442]
As above, nearly like in the standalone. 
- [ ] Ability to pick apples
- [ ] Ability to pick pumpkins
- [ ] Apple model+texture
- [ ] Pumpkin model+texture

**Comment by CoftSock** [comment]
youll need apple models but then once they're done it wouldnt be hard

**Comment by ghost** [comment]
Is there anyone we can ask? Somebody who did models before?

**Comment by R4Z0R49** [comment]
Plan is to add both apples and pumpkins at some point

**Comment by ghost** [comment]
Alright, I changed the title and edited the first post. But there should be definitly a lower chance to find "good" apples and pumpkins. Maybe we should add rotten versions of both aswell and then add chances like:

2/4 to find nothing
1/4 to find a rotten
1/4 to find a fresh one

Or that some trees just gives you rotten ones and if you eat them there is a chance of getting infected.This should make the whole thing more real and dynamic.

**Comment by ghost** [comment]
I have got at least an apple model - it just needs some textures. 

![mod_apple](https://cloud.githubusercontent.com/assets/3810930/7613747/c6886b8c-f992-11e4-81e3-586484cdd984.JPG)

**Comment by ghost** [comment]
![apple_finished](https://cloud.githubusercontent.com/assets/3810930/7629133/94630c32-fa2a-11e4-8c9a-2d232c7e0511.JPG)

I know the texture doesnt look perfect, but I guess its enough to get into the game. 
What do you say @R4Z0R49

**Comment by CoftSock** [comment]
you will also need a picture that shows up when the item is in the Gear UI

**Comment by ghost** [comment]
Here it is: 

![apple_icon](https://cloud.githubusercontent.com/assets/3810930/7636326/ed29a09c-fa66-11e4-9ede-260512eb8b19.png)

![apple_finished](https://cloud.githubusercontent.com/assets/3810930/7636293/b43e2fd2-fa66-11e4-94a4-a6f5caa06b28.JPG)

I dont know the extact size of the gear icons so somebody of you needs to resize and convert them to PAA. And I need finally to export this thing to P3D and to check whether the relation of the apple's size is correctly.

btw. please remember that I'm totally new to modeling and texturing!

**Comment by KPanicEKS** [comment]
How many polys?

**Comment by ghost** [comment]
I have no idea! How to find out and how to count? Are there any other things I should take a look at or is there any way I can send you the whole model with textures?

**Comment by AsFoxy** [comment]
It's easy to get rid of polys. If you need any help bringing it over to Arma let me know.

**Comment by ghost** [comment]
@AsFoxy I have send you an email.

**Comment by CoftSock** [comment]
256 X 256 seems to be the picture size for the item for the Gear UI

**Comment by ghost** [comment]
So finally, here is the gear picture in a size of 256x256. @AsFoxy takes a look at my model and will upload three versions with different polycount here. 

![apple_icon](https://cloud.githubusercontent.com/assets/3810930/7650643/d8403820-faf8-11e4-8f78-f99d94225883.png)

**Comment by CoftSock** [comment]
did you also make a rotten texture one?

**Comment by HeroSmurf** [comment]
Let's approach rotten like a failed fishing cast. 

When you recieve a rotten apple you instantly throw it away. 
"The apple you found turns out to be rotten"

**Comment by KPanicEKS** [comment]
ya, i have no interest in hanging onto rotten fruit.

**Comment by ghost** [comment]
I thought about giving only a few trees the chance to find good apples. And if you are starving and you only find rotten ones you will need to eat them with a chance of getting infected.

**Comment by R4Z0R49** [comment]
Its not really going to matter you wont be able to attach anything to trees all we will be doing is looking for the tree model and running a script to give a good apple or bad apple based on chance

**Comment by ghost** [comment]
Mmh. Well I thought that the server would check all apple tree objects and decides for each id whether the tree has apples, rotten or none. 

We could add the rotten ones anyway to spawn them in buildings. Just in case someone is starving and is frustrated from looking for food and so he needs to decide, eat the rotten and may get infected or die in the next minutes. I think decisions like that makes Dayz to something special.

**Comment by HeroSmurf** [comment]
Yeah but that's assuming rotten apples, what I think you're suggesting would be a food stopgap, will be more prevalent in buildings than the food we're already spawning in. 

I still don't agree that food is hard to find (not something you've asserted, just trying to relate why I LIKE the idea of finding fresh fruit and DISLIKE the idea of finding rotten fish/fruit/animals)

Let's just put them on the ground in simple spawn points next to orchard trees (without zombie spawns on top of them LOL) and be done with it. I'd hate to see the work you did on this not get into the game, makes sense, adds to the environmental feel of the game. 

An apple a day keeps the zombies away

**Comment by R4Z0R49** [comment]
i think we might already have a pumpkin model.
